Makefile

```diff
@@ -0,0 +1,70 @@
+#!make
+
+# Include config variables for custom course
+include envfile
+export $(shell sed 's/=.*//' envfile)
+
+# Execute commands regardless of physical state of files
+.PHONY: build-docs build-dev dev dcourse lcourse  pltest lint unit test docs clean node  
+
+# Install node dependencies (requires node > 10.0)
+node: 
+	npm ci
+
+# Generates a development docker from current directory
+build-dev:
+	docker build -t pltestlocal .
+
+# Show the development version of the prairielearn
+dev: build-dev
+	docker run -it --rm -p 3000:3000 pltestlocal
+
+dcourse: build-dev
+	docker run -it --rm -p 3000:3000 -v ${COURSE_REPO_PATH}:/course prairielearn/prairielearn
```

Comment:
Should this run `pltestlocal` rather than `prairielearn/prairielearn`?

Reply:
Yes. Good catch!